lntest/itest/lnd_multi-hop_htlc_local_timeout_test.go

```diff
@@ -143,20 +153,39 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest,
 
 	// With the closing transaction confirmed, we should expect Bob's HTLC
 	// timeout transaction to be broadcast due to the expiry being reached.
-	htlcTimeout, err := waitForTxInMempool(net.Miner.Node, minerMempoolTimeout)
+	// If there are anchors, we also expect Carol's anchor sweep now.
+	txes, err := getNTxsFromMempool(net.Miner.Node, expectedTxes, minerMempoolTimeout)
 	if err != nil {
 		t.Fatalf("unable to find bob's htlc timeout tx: %v", err)
 	}
+	var htlcTimeout *chainhash.Hash
```

Comment:
add a comment what this loop is achieving

Reply:
added